_grab_ui.py

```diff
@@ -677,7 +692,13 @@ def ospec() -> str:
 
 
 --title
-(Internal)'''
+(Internal)
+
+
+--copy-to
+dest=copy_to
+type=str
+Copy to: 'clipboard' or 'primary'/selection or 'secondary' buffer'''
```

Comment:
I think it makes more sense to document the user-facing option above the internal ones.

Reply:
Sorry, I'm new in your project, in the Kitty and in the Python too ;-)
So, where do you suggest to put the help? In the grab.py? I don't see a way to make it visible with a `kitten @ kitten kitty_grab/grab.py --help` except to parse --help option by myself.